src/core/simulation_results.jl

```diff
@@ -365,35 +366,49 @@ function RealizedMeta(
     initial_time::Union{Nothing, Dates.DateTime} = nothing,
     len::Union{Int, Nothing} = nothing,
 )
+    existing_timestamps = get_existing_timestamps(res)
+    interval = existing_timestamps.step
+    resolution = PSY.get_time_series_resolution(get_system(res))
+    interval_len = Int(interval / resolution)
```

Comment:
This will throw an error if there is a remainder as is:
```
julia> Int(1.0/2.0)
ERROR: InexactError: Int64(0.5)
Stacktrace:
 [1] Int64(::Float64) at ./float.jl:710
 [2] top-level scope at REPL[6]:1
 ```
 
 Should I add an assert statement anyway?

Reply:
Right, no, that's sufficient.